lexos/static/js/scripts_upload.js

```diff
@@ -1,197 +1,197 @@
 $(function () {
-  if ($('.fa-folder-open-o')[0].style.display != 'none' &&
-    $('.fa-folder-open-o')[0].innerText.indexOf('Response') != -1) {
-    alert('Steven\'s special bug detected! Click OK and the page will load.')
-    $('.fa-folder-open-o')[0].style.display = 'none'
-  }
-
+  /*message for when you hover on the open folder icon on the top right*/
   $('[data-toggle="tooltip"]').tooltip()
-
   $('#uploadbrowse').click(function () {
     $('#fileselect').click()
   })
 
-  // ------------------- FILEDRAG -----------------------------
-
-  var allowedFileTypes = ['txt', 'xml', 'html', 'sgml', 'lexos']
-
+  //alias function
   function $id(id) {
     return document.getElementById(id)
   }
 
-  function AllowedFileType(filename) {
-    var splitName = filename.split('.')
-    var fileType = splitName[splitName.length - 1]
-    if ($.inArray(fileType, allowedFileTypes) > -1) {
-      return true
-    } else {
-      return false
-    }
-  }
-
-  // file drag hover
-  function FileDragHover(e) {
-    e.stopPropagation()
-    e.preventDefault()
-    e.target.className = (e.type == 'dragover' ? 'hover' : '')
-  }
-
-  var numberOfFileDone = parseInt($('.fa-folder-open-o')[0].id)
-
+  let numberOfFileDone = parseInt($('.fa-folder-open-o')[0].id)
```

Comment:
If you use let, please inform me in code comments the following:
1. why do you have to mutate the variable
2. where in the code did you mutated the variable.